### DISCUSSION TITLE
Reflect current search params in the URL or add a "Share Search" Button

### DISCUSSION THREAD

**Discussion by RealJohnnyTime** [solodit/feedback#60] [New Feature Requests]
Currently, it's not possible to share search results with others because the search parameters are not reflected in the URL.

In many cases, it would be very useful to send filtered results or save the link for later access. It would be great if the search parameters were included in the URL, or if there was a "Share Search" button that could generate a link leading directly to the filtered issues.

**Comment by hans-cyfrin** [comment]
@RealJohnnyTime thanks for the suggestion, it is in our backlog and we will implement in the near future.
